build.sbt

```diff
@@ -95,30 +58,15 @@ lazy val microservice = Project(appName, file("."))
   .settings(
     scalacOptions ++= Seq(
       "-Wconf:cat=unused&src=views/.*\\.scala:s",
-      "-Wconf:cat=unused&src=.*RoutesPrefix\\.scala:s",
-      "-Wconf:cat=unused&src=.*Routes\\.scala:s",
-      "-Wconf:cat=unused&src=.*ReverseRoutes\\.scala:s"
+      // https://www.scala-lang.org/2021/01/12/configuring-and-suppressing-warnings.html
+      // suppress warnings in generated routes files
+      "-Wconf:src=routes/.*:s"
     )
   )
-  .disablePlugins(sbt.plugins.JUnitXmlReportPlugin)
-
-
-lazy val AcceptanceTest = config("acceptance") extend Test
-lazy val SandboxTest = config("sandbox") extend Test
 
-coverageMinimumStmtTotal := 77
-coverageFailOnMinimum := true
-coverageExcludedPackages := Seq(
-  "<empty>",
-  ".*definition.*",
-  "prod",
-  "testOnlyDoNotUseInAppConf",
-  "app",
-  "uk.gov.hmrc.BuildInfo"
-).mkString(";")
 
 commands ++= Seq(
-  Command.command("run-all-tests") { state => "test" :: "it:test" :: "acceptance:test" :: "sandbox:test" :: state },
+  Command.command("run-all-tests") { state => "test" :: state },
```

Comment:
Do we need all the testAll/cleanAll etc

Reply:
I thought of adding them, then wondered what the value would be. I could, for consistency but no other reason.

Reply:
Just consistency is good.